cn.navclub.vdbfx.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "cn.navclub.dbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "v-dbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libgee",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/libgee.git",
+                    "commit": "4ed45f6f32dcb8712501d38a44ec754d4937a26e"
+                }
+            ]
+        },
+        {
+            "name": "sqlite",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure --prefix=/app",
+                "make",
+                "make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.sqlite.org/snapshot/sqlite-snapshot-202205171511.tar.gz",
+                    "sha1": "90d18bd735f96111df61c4dc78c1f011ac95ad1b"
+                }
+            ]
+        },
+        {
+            "name": "flatpak_mysql",
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "lib",
+                "include"
```

Comment:
Looking further it looks like this doesn't do what you think it is doing.

Just to clarify this will cleanup the *installed* files. It's not about `make clean`